test/twine_test.coffee

```diff
@@ -277,10 +277,10 @@ suite "Twine", ->
       node = setupView(testView, context = fn: @spy())
       data = {test: 'bla123'}
 
-      $(node).trigger 'submit', data
```

Comment:
While the lib shouldn't use jQ, I think it's fair game to use it in the tests if that makes the tests pass.  Does your troublesome test pass if you use jQ as the triggerer?